### DISCUSSION TITLE
About deployment

### DISCUSSION THREAD

**Discussion by jensenw-pki** [arc53/DocsGPT#693] [Q&A] [answered]
Sorry, I am a beginner of GPT.
I can see there is docker option to deploy the DocsGPT on some machine with providing OPEN AI key. does it means once I deploy it, i can get the same capability as  https://docsgpt.arc53.com/?
And I don't need to train the model with GPU listed in the [Our Open-Source models optimized for DocsGPT] section, Am I right? (https://github.com/arc53/DocsGPT#our-open-source-models-optimized-for-docsgpt)

**Comment by larinam** [comment, marked as answer]
Hi, you don't need a GPU to run DocsGPT with OpenAI.
But please follow the guide on how to run it locally: https://github.com/arc53/DocsGPT#quickstart
